python/ray/tune/experiment.py

```diff
@@ -206,8 +210,7 @@ def checkpoint_dir(self):
 
     @property
     def remote_checkpoint_dir(self):
-        if self.spec["upload_dir"]:
-            return os.path.join(self.spec["upload_dir"], self.name)
+        return self.spec["upload_dir"]
```

Comment:
this is wrong - the remote checkpoint dir is `os.path.join(self.spec["upload_dir"], self.name)`

Reply:
The checkpoint directory corresponds to the directory of a single experiment, not the upload_dir, which is one-to-one with the local_dir (which contains multiple expmts)

Reply:
I'm actually not sure why you changed this in the first place though?

Reply:
my bad, fixed. changed because i need to pass in `remote_checkpoint_dir` to the spec so that I can pass it into `Trial`, which in turn can create its own directory within `remote_checkpoint_dir`